### PR TITLE
surrogate metal for ACAT in analyze_slab() added

### PR DESCRIPTION
**Pull request by sakim8048**
## Describe your changes
Added ACAT related keywords in main.py:
tol_acat : this is a tolerance keyword for ACAT slabAdsorptionSite(). Now tolerance can be adjusted by users in case identified sites via ACAT is not satisfied. 
surrogate_metal_acat: Also ACAT slabAdsoprtionSite() keyword. if self.metal is one of the transition metals in EMT calculator, surrogate_metal_acat will be the same as metal keyword. Otherwise, user has to pick one that is the most appropriate for the system. 
scale : initial_magnetization for QE keyword. 

## Functionality
in main.py, def analyze_slab(),
emt_metals = [Ni, Cu, Pd, Ag, Pt, Au]
if metal is not one of those emt_metal list, surrogate_metal_acat keyword will be used as a surrogate metal for ACAT calculator. 

## Note
This  can be merged to master. If not ready to be added to master branch yet, I can submit a new pull request for metal_improvement branch.

### CODE REVIEW

**Review by mjohnson541 — CHANGES_REQUESTED** [review]
I have a few comments.

File: pynta/main.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 class Pynta:
     def __init__(self,path,rxns_file,surface_type,metal,label,launchpad_path=None,fworker_path=None,
         vacuum=8.0,repeats=(3,3,4),slab_path=None,software="Espresso",socket=False,queue=False,njobs_queue=0,a=None,
+        scale=0, #for initial_magnetization. if scale=0, there is no magnetization applied.
```
[the remark this answers is not in the document]
Comment (sakim8048) [reply pynta/main.py:28]: It is true that we don't want to be acat specific. You are right, we should store all the variables in an acat specific dictionary.
I can change the default to self.metal if that is more convenient. 
Tolerance for ACAT default is 0.5, which is the default value I set as but can be or should be adjusted depending on the system. It is also documented in the ACAT code that if the user is not satisfied with the determined sites, tolerance should be adjusted. I still think it is better to have an option to adjust the tolerance.

File: pynta/main.py [hunk 2]
```diff
@@ -58,7 +61,10 @@ def __init__(self,path,rxns_file,surface_type,metal,label,launchpad_path=None,fw
         self.metal = metal
         self.adsorbate_fw_dict = dict()
         self.software_kwargs = software_kwargs
-
+        self.scale = scale
```
[the remark this answers is not in the document]
Comment (sakim8048) [reply pynta/main.py:64]: will take it off.

File: pynta/main.py [hunk 3]
```diff
@@ -148,9 +154,18 @@ def generate_slab(self,skip_launch=False):
...
     def analyze_slab(self):
         full_slab = self.slab
-        cas = SlabAdsorptionSites(full_slab, self.surface_type,allow_6fold=False,composition_effect=False,
-                        label_sites=True,
-                        surrogate_metal=self.metal)
+        emt_metals = ['Ni','Cu','Pd','Ag','Pt','Au'] #list of pure transition metals used in EMT calculator
+        
+        print('Input metal element is:', self.metal,' Surrogate metal option for ACAT SlabAdsorptionSite() is set as:',self.surrogate_metal_acat) #double check 
+
+        for metals in emt_metals:
```
[the remark this answers is not in the document]
Comment (sakim8048) [reply pynta/main.py:161]: 1,2: surrogate_metal_acat will be set as None as we discussed.
3: Task.py will be fixed accordingly.

However, until we have preprocessing notebook, let's put this PR on the side.

**Pull request closed without merging**